punchbowl/levelq/pca.py

```diff
@@ -129,15 +225,20 @@ def find_bodies_in_image(frame: str | NDCube | WCS) -> list:
         location = frame.meta["GEOD_LON"].value, frame.meta["GEOD_LAT"].value, frame.meta["GEOD_ALT"].value
         wcs = frame.wcs
         image_shape = frame.data.shape
-    elif not isinstance(frame, WCS):
+    elif isinstance(frame, tuple):
+        meta, wcs = frame
+        location = meta["GEOD_LON"].value, meta["GEOD_LAT"].value, meta["GEOD_ALT"].value
+        image_shape = wcs.array_shape
+    else:
         msg = "Type of 'frame' not recognized"
         raise TypeError(msg)
 
     results = []
     for body in ["Mercury", "Venus", "Moon", "Mars", "Jupiter", "Saturn"]:
         body_loc = get_body(body, time=Time(wcs.wcs.dateobs), location=EarthLocation.from_geodetic(*location))
         x, y = wcs.world_to_pixel(body_loc)
-        w = 40 if body == "Moon" else 10
+        w = 100 if body == "Moon" else 10
```

Comment:
The logic here being that the moon is big? Could you document this number with a comment?